packages/prisma/schema.prisma

```diff
@@ -43,9 +43,10 @@ enum PeriodType {
 }
 
 enum CreationSource {
-  API_V1 @map("api_v1")
-  API_V2 @map("api_v2")
-  WEBAPP @map("webapp")
+  API_V1           @map("api_v1")
+  API_V2           @map("api_v2")
+  WEBAPP           @map("webapp")
+  SELF_SERVE_ADMIN @map("self_serve_admin")
```

Comment:
We should expand on the `CreationSource` to pinpoint where watchlisted users are created